profile/README.md

```diff
@@ -0,0 +1,29 @@
+[![logo](https://raw.githubusercontent.com/wmde/.github/main/images/WMDEGitHubBanner.png)](https://github.com/wmde)
+
+Wikimedia Deutschland is the German chapter of the global [Wikimedia movement](https://www.wikimedia.org/). Our technical focus is on [Wikidata](https://www.wikidata.org/), the [Wikibase ecosystem](https://wikiba.se/) and software that serves the German speaking and wider Wikimedia community.
+
+## Our projects
+
+### Wikidata
+
+A collaboratively edited multilingual knowledge graph that powers Wikipedia, Wikivoyage, Wiktionary, Wikisource, and many other sites and services beyond just Wikimedia projects! See [wikidata.org](https://www.wikidata.org).
+
+### Wikibase Suite
+
+Dockerized deployment of Wikibase, the technology behind Wikidata. It alows you to host your very own linked open data knowledge base. See [installation guide on MediaWiki](https://www.mediawiki.org/wiki/Wikibase/Suite).
+
+### Wikibase Cloud
+
+A fully managed Wikibase instance that provides the support needed for those who want to have their own knowledge graph quickly and easily. See [wikibase.cloud](https://www.wikibase.cloud).
+
+### Fundraising Tech
+
+Software that powers Wikimedia Deutschland's fundraising efforts including apis, user interfaces, banners and more. See [spenden.wikimedia.de](https://spenden.wikimedia.de/).
```

Comment:
"apis" vs "APIs" was the only other think I spotted. Not sure if it is intentional or just a copy-paste from casual chat style. That's all, done. :-)

Reply:
Ok, thank you! I'll check to see if @datn has a moment to go over it all and then we hopefully will be good to go :blush:

Reply:
Line 17: reword: "A fully managed Wikibase instance made available quickly and easily for those who want their own knowledge graph". 
Line 21: change to "APIs"; insert serial comma here or remove from line 9 above

Reply:
Didn't see [this comment](https://github.com/wmde/.github/pull/1#discussion_r1274882449), but the changes for that have gone through in fa7f521 :upside_down_face: